docs/backup-restore.md

```diff
@@ -1,134 +1,204 @@
-# A Guide on How to Backup and Restore Databases Used Through ScalarDB
+# How to Back Up and Restore Databases Used Through ScalarDB
 
-Since ScalarDB provides transaction capability on top of non-transactional (possibly transactional) databases non-invasively, you need to take special care of backing up and restoring the databases in a transactionally-consistent way.
-This document sets out some guidelines for backing up and restoring the databases that ScalarDB supports.
+Since ScalarDB provides transaction capabilities on top of non-transactional (possibly transactional) databases non-invasively, you need to take special care to back up and restore the databases in a transactionally consistent way.
```

Comment:
The phrase `possibly transactional` here and having it in parenthesis is a bit confusing. Can we change this sentence as follows?:

```suggestion
Since ScalarDB provides transaction capabilities on top of non-transactional or transactional databases non-invasively, you need to take special care to back up and restore the databases in a transactionally consistent way.
```

Reply:
Sounds better!